### PR TITLE
Updated README.md to add installation instruction for Python shared l…

### PR DESCRIPTION
**Pull request by santokalayil**
…ibrary on Fedora

Thank you for contributing to pyo3!

Please consider adding the following to your pull request:
 - an entry for this PR in newsfragments - see [https://pyo3.rs/main/contributing.html#documenting-changes]
 - docs to all new functions and / or detail in the guide
 - tests for all new or changed functions

PyO3's CI pipeline will check your pull request. To run its tests
locally, you can run ```cargo xtask ci```. See its documentation
 [here](https://github.com/PyO3/pyo3/tree/main/xtask#readme).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -130,6 +130,12 @@ To install the Python shared library on Ubuntu:
 sudo apt install python3-dev
 ```
 
+To install the Python shared library on Fedora:
+
+```bash
+sudo dfn install python3-devel
```
Comment (birkenfeld) [inline README.md:136]: Should be `dnf` I think.

File: README.md [hunk 2]
```diff
@@ -130,6 +130,12 @@ To install the Python shared library on Ubuntu:
 sudo apt install python3-dev
 ```
 
+To install the Python shared library on Fedora:
```
Comment (birkenfeld) [inline README.md:133]: I think it can be generalized to "RPM-based distributions"?

**Review by birkenfeld — CHANGES_REQUESTED** [review]
Thanks for the PR! The test failure should disappear after a rebase.

File: README.md (hunk 2 above)
Comment (adamreichold) [reply README.md:133]: I think this is complicated. SUSE uses `zypper`, the still supported RHEL 7 uses `yum`. The package name seems to be pretty universal though.

Comment (birkenfeld) [reply README.md:133]: Yeah, I meant that it should then say "install the package `python3-devel`".

File: README.md (hunk 1 above)
Comment (santokalayil) [reply README.md:136]: Yes.. I mistyped it

File: README.md (hunk 2 above)
Comment (birkenfeld) [reply README.md:133]: OK, so let's say
```suggestion
To install the Python shared library on RPM based distributions (e.g. Fedora, Red Hat, SuSE), install the `python3-devel` package.
```
and remove the `dnf` snippet below.

**Review by birkenfeld — DISMISSED** [review]

**Comment by davidhewitt** [comment]
Thanks! I've squashed and will merge.

bors r+

**Pull request merged**